traefik/templates/rbac/role.yaml

```diff
@@ -31,7 +31,9 @@ rules:
     resources:
       - services
     verbs:
+      {{- if $.Values.hub.token }}
       - get
+      {{- end }}
```

Comment:
Why adding a conditionnal here ?
In upstream 3.1 rbac [reference](https://raw.githubusercontent.com/traefik/traefik/v3.1/docs/content/reference/dynamic-configuration/kubernetes-crd-rbac.yml), this get is needed
Same with rbac [reference](https://raw.githubusercontent.com/traefik/traefik/v3.1/docs/content/reference/dynamic-configuration/kubernetes-gateway-rbac.yml) for gateway api
Sa